src/shared/ui/components/widget-tab/tab-handle.component.tsx

```diff
@@ -0,0 +1,39 @@
+import { classes } from '../../utils/classes';
+
+import { TabHandleProperties } from './tab-handle.types';
+
+export const TabHandle = ({
+  left,
+  imageSrc,
+  name,
+  isActive,
+  theme,
+  onClick,
+}: TabHandleProperties) => {
+  return (
+    <div
+      onClick={onClick}
+      style={{
+        left,
+      }}
+      className={classes(
+        'absolute -top-[25px] flex h-[25px] w-[100px] cursor-pointer flex-row items-center gap-[6px] rounded-t-lg pl-2 pt-1 font-bold transition-all duration-75 ease-linear',
+        {
+          'bg-[#2d2d2d]': theme === 'dark',
+          'bg-white': theme === 'bright',
+          '-top-[23px] -translate-x-[3px] scale-95 brightness-75': !isActive,
+        },
+      )}
+    >
+      {imageSrc && <img className="h-3/4" src={imageSrc} />}
```

Comment:
I am thinking if it would be better to hardcode height in px values, not sure.
I am imaging a situation that tab will grow in height one day and should image grow too?

I am actually find with both, so it's up to you :)

Reply:
updated, fixed height seems to be safer :D